src/bindings/python/src/openvino/frontend/pytorch/decoder.py

```diff
@@ -389,6 +391,16 @@ def input_is_none(self, index: int) -> bool:
                     return pt_value is None
         return False
 
+    def may_produce_alias(self, in_index: int, out_index: int) -> bool:
+        if self.get_op_type() in ["aten::conv1d", "aten::conv2d", "aten::conv3d"]:
+            # AliasDB::may_contain_alias sometimes return True for tensors produced by convnd, we have to workaround that
```

Comment:
strange moment. Why is it valid for these particular operations? It can be a case for other operations then

Reply:
It can be and this is really confusing, but currently I saw this happen on conv2d only